packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -179,16 +215,18 @@ class Site extends React.Component<ISiteProps & IManagedClasses<ISiteManagedClas
             }
         });
     }
-
+    /* tslint:disable */
     private getPaneCollapseToggle(): JSX.Element {
         return (
             <button
                 onClick={this.handlePaneCollapse}
                 className={this.props.managedClasses.site__paneToggleButton}
-                dangerouslySetInnerHTML={{__html: glyphGlobalnavbutton}}
-            />
+            >
+                <div className={this.props.managedClasses.site__paneToggleButtonIcon} dangerouslySetInnerHTML={{__html: glyphGlobalnavbutton}}/>
```

Comment:
The HTML5 spec doesn't allow for a div to be wrapped inside a button - let's use a span instead.